dockerfiles/FluentdLoggingContainerDockerfile

```diff
@@ -1 +1,28 @@
-# Currently, we are using a default Fluentd image from Dockerhub. Needs to be customized by Clipper
\ No newline at end of file
+FROM fluent/fluentd:v1.5-debian-1
```

Comment:
Also, the docker container need to be built. See something like 
https://github.com/ucbrise/clipper/blob/df96eb4924809fedc5a70ef1bc080129c2bf962c/bin/shipyard/clipper_docker.cfg.py#L93

Reply:
Not sure if my fix will resolve this part. Let's see if our CI can pass